Source/JavaScriptCore/runtime/JSGlobalObject.h

```diff
@@ -924,6 +924,7 @@ class JSGlobalObject : public JSSegmentedVariableObject {
     RegExpGlobalData& regExpGlobalData() { return m_regExpGlobalData; }
     static ptrdiff_t regExpGlobalDataOffset() { return OBJECT_OFFSETOF(JSGlobalObject, m_regExpGlobalData); }
 
+    static ptrdiff_t offsetOfGlobalThis() { return OBJECT_OFFSETOF(JSGlobalObject, m_globalThis); }
```

Comment:
this seems unused

Reply:
It is used in DFGSeculativeJIT.cpp's L8593.